### PR TITLE
Added more snippets

### PR DESCRIPTION
**Pull request by lgarcin**

### CODE REVIEW

File: snippets/language-restructuredtext.cson [hunk 1]
```diff
@@ -1,10 +1,139 @@
...
+  'sidebar directive':
+    'prefix': 'sidebar'
+    'body': '.. sidebar:: ${1:title}\n\n\t$0'
+  'table directive':
+    'prefix': 'table'
+    'body': '.. table:: ${1:title}\n\n\t$0'
+  'tip directive':
+    'prefix': 'tip'
+    'body': '.. tip::\n\n\t'
+  'topic directive':
```
Comment (mshenfield) [inline snippets/language-restructuredtext.cson]: Would you consider adding this snippet for the [`toctree` directive](http://www.sphinx-doc.org/en/stable/markup/toctree.html#toctree-directive)
```cson
  'toctree directive':
    'prefix': 'toctree'
    'body': '.. toctree::\n\t:maxdepth: ${1:2}\n\n\t$0'
````

Comment (Alhadis) [reply snippets/language-restructuredtext.cson]: 👆 What he said.

**Comment by lgarcin** [comment]
Done. Thanks for the review.

**Comment by Alhadis** [comment]
Thanks!

I'll strive to get a release cut as soon as possible, which is easier said than done since my MacBook died the other night, and I'm currently Atom-less...😣

**Pull request merged**